src/universal/mutations/CreateReflectionMutation.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Creates a reflection for the retrospective meeting.
+ *
+ * @flow
+ */
+import {commitMutation} from 'react-relay';
+import {Environment, RecordProxy, RecordSourceSelectorProxy} from 'relay-runtime';
+
+import clientTempId from 'universal/utils/relay/clientTempId';
+import makeEmptyStr from 'universal/utils/draftjs/makeEmptyStr';
+
+type Variables = {
+  content?: string,
+  meetingId: string,
+  retroPhaseItemId: string,
+  sortOrder: number
+};
+
+type CompletedHandler = (response: ?Object, errors: ?Array<Error>) => void;
+
+type ErrorHandler = (error: Error) => void;
+
+graphql`
+  fragment CreateReflectionMutation_team on CreateReflectionPayload {
+    meeting {
+      id
+    }
+    reflection {
+      content
+      id
+      isViewerCreator
+      retroPhaseItemId
+      sortOrder
+    }
+}
+`;
+
+const mutation = graphql`
+  mutation CreateReflectionMutation($content: String, $meetingId: ID!, $retroPhaseItemId: ID!, $sortOrder: Float!) {
+    createReflection(content: $content, meetingId: $meetingId, retroPhaseItemId: $retroPhaseItemId, sortOrder: $sortOrder) {
+      ...CreateReflectionMutation_team @relay(mask: false)
+    }
+  }
+`;
+
+export const createReflectionUpdater = (payload: RecordProxy, store: RecordSourceSelectorProxy) => {
+  // TODO - https://github.com/dan-f/maeby
```

Comment:
my solution to this is 2-fold:
first, i make sure the payload is truthy in the updater below (but doing it here doesn't hurt!)
then, i use `getInProxy`. since relay has a wonky API, `getInProxy` has to take you to a primitive. eg `meetingId = getInProxy(payload, 'meeting', 'id')`